code/Game.cs

```diff
@@ -1,11 +1,12 @@
 ﻿using System;
 using Sandbox;
 using System.Collections.Generic;
+using System.Linq;
 using System.Threading.Tasks;
 
 namespace TTTGamemode
 {
-    [ClassLibrary("ttt-reborn", Title = "Trouble in Terry's Town")]
+    [Library("ttt", Title = "Trouble in Terrorist Town")]
```

Comment:
Why do you change the name, we've chosen?

Reply:
Having a dash in the library name actually stops the project from launching completely, so trimmed off the `-reborn` part. Let me know if `tttreborn` maybe looks better.

The `.addon` folder has the title `Trouble in Terrorist Town` which is what users see, wanted to keep it consistent so changed it from `Trouble in Terry's Town` to `Trouble in Terrorist Town`
https://github.com/TTTReborn/ttt-reborn/blob/bc2819e1cde422a1f2b729a45e0572041efc56ea/.addon#L20

Let me know and I can update both copies to whatever the reborn team sees fit.

Reply:
Well as far as Im informed we want to go full-in with ttt reborn and Terry's Town to be more "modern" and give youtubers the possibility to not get demonetized for writing terrorist in their title.
So I guess you could change that already, but you can also wiat for more approval of the core developers.

Reply:
Yep, what @ZenBre4ker says is correct. We want `Trouble in Terry's town`. It still has the abbrevation TTT, but the name is slightly different while also fitting to S&Box.

Reply:
Agreed. Additionally, I'd like to use "tttreborn" instead of just "ttt" to avoid issues with other TTT projects for S&Box as we can't claim the exact name ours imo (regarding BKU's agreement).